app/controllers/search/results_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Search::ResultsController < ApplicationController
+
+  def index
+    @obituaries_api = ObituarySearch.new.advanced_query(params) || []
```

Comment:
what does this "|| []" do?

Reply:
This is something I added when I was helping debug. If `ObituarySearch.new.advanced_query(params)` comes back as `nil` it will send an empty array to the view instead of `nil` because that was giving us an error in the view.